plugins/polio/api/campaigns/campaigns.py

```diff
@@ -960,17 +960,26 @@ def csv_campaign_scopes_export(self, request, **kwargs):
         round = Round.objects.get(pk=request.GET.get("round"))
         campaign = round.campaign
         org_units_list = []
-        org_units = campaign.get_districts_for_round(round)
```

Comment:
I wonder if it's worth keeping these methods in the model since they assume a lot of stuff has already been prefetched